COMPILE.md

```diff
@@ -41,6 +41,18 @@ It's better to define the CMake Build type, preferably `RelWithDebInfo`, that de
 
     An Android.mk is provided that should compile with an NDK
 
+*iOS*
+---
+
+```
+mkdir build
+cd build
+cmake .. -G Xcode -DCMAKE_TOOLCHAIN_FILE={where ios.toolchain.cmake is located} -DNOX11=ON -DNOEGL=ON -DSTATICLIB=ON -DPLATFORM=OS
```

Comment:
isn't `-DPLATFORM=OS` a typo ? not iOS or OSX?

Reply:
not a typo, the unofficial iOS cmake toolchain uses OS for iOS devices, SIMULATOR for iOS simulators

Reply:
Ok, my bad :)